.github/workflows/merge-prometheus.yaml

```diff
@@ -27,10 +27,15 @@ jobs:
         go.sum
         .golangci.yml
       assets-cmd: |
-        make assets-compress
-        find web/ui/static -type f -name '*.gz' -exec git add {} \;
-        git add web/ui/embed.go
-        git diff --cached --exit-code || git commit -s -m "[bot] assets: generate"
+        # in case we change the upstream or upstream-branch variable value
```

Comment:
do you mean `downstream` and `dowstream-branch` as the comparison should be done locally after the pull.

(No github actions expert, but couldn't we use `${{ inputs.downstream-branch }}` in here?)

Reply:
Yes you are right, the comment was wrong. I updated it.

> (No github actions expert, but couldn't we use ${{ inputs.downstream-branch }} in here?)
We could (using `env:` at the job level) but I think this would complicate things. we'd have to either match the value to a git remote name or do some string massaging to make the `downstream` input into a url. I thought is was better to just rely on gits remote naming.

Reply:
Don't know if we could use `origin/${{ inputs.downstream-branch }}` to be future proof, but lgtm

Reply:
(missed your response as it was rendered as a quote)

Reply:
sorry thats my bad :(